pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -744,3 +747,140 @@ func TestOnDNSResponse(t *testing.T) {
 		})
 	}
 }
+func TestFQDNCacheMinTTL(t *testing.T) {
```

Comment:
I feel like in theory we only need to test `parseDNSResponse`, because this is the only place where `minTTL` is actually used. We just need to make sure that `minTTL` can override the TTL included in the DNS response. cc @Dyanngg 

However, I don't feel super strongly about it, so if others think it is better to test `onDNSResponseMsg` "end-to-end", it is fine by me.

Reply:
I am awaiting a comment on this one to implement the nit change asked above in previous comment , moreover, i had also pushed the commit below [refactor to test just the parseDNSResponse](https://github.com/antrea-io/antrea/pull/6808/commits/6c6dcc81036692a72d7301983bf86c6c3d4d8a57) considering the feedback.